src/index.js

```diff
@@ -2,5 +2,8 @@ import React from 'react';
 import { render } from 'react-dom';
 import App from './containers/App';
 import './assets/scss/base.scss';
+import decodeToken from './helpers/validationHelpers/decodeToken';
 
-render(<App />, document.querySelector('#root'));
+const user = decodeToken() ? decodeToken() : null;
```

Comment:
@Danielshow I don't get what's going on here.

Reply:
@ef, I have fixed it, just checked now and `decodedToken` returns null if the token does not exist.